### ISSUE TITLE
API with RDC only in our remote website

### ISSUE THREAD

**Issue by kpetrow** [Ylianst/MeshCentral#3448]
Love this product and we are going to adopt it into our system.  I realize I could start digging through the code and saw the url append trace=1 to view websocket traffic.  But was hoping someone might have a repo already done.

We basically are looking to replace NoVNC that runs iframe or popup that routes through a vnc repeater.  We would use the meshcentral server as our repeater/connector and hopefully have a replacement for the Novnc Client.

Basically looking for the code to extract out this portion of meshCentral.
![image](https://user-images.githubusercontent.com/18248938/149260669-290f026c-0ebc-4116-b898-52ba8647ac67.png)

Thanks again

**Comment by Ylianst** [comment]
Take a look at section 14 of the [MeshCentral User's Guide](https://meshcentral.com/info/docs/MeshCentral2UserGuide.pdf). You can add `?hide=255` to the URL to hide everything except the screen. Let me know if that works.

Another way to go is to generate a guest sharing URL for a device and embed that in an iframe.

Let me know if that works.

**Comment by kpetrow** [comment]
Thank you for the prompt response i will look into this, and respond in next 24 hours

**Comment by kpetrow** [comment]
Thank you!.  RTFM would have been appropriate reply, but this is exactly what  was looking for.

Happy New year!

**Comment by kpetrow** [comment]
How do you fetch NodeID?
![image](https://user-images.githubusercontent.com/18248938/149342939-96019c63-a47e-478c-b5f6-f7cb92bda5bb.png)

**Comment by Ylianst** [comment]
You can click on a device on the web page and look at the URL above...

![image](https://user-images.githubusercontent.com/1319013/149571693-4a46e462-15fa-49d0-978c-eb24e66518ea.png)

or use 'meshctrl' to list all devices in an account, [documentation is here](https://meshcentral.com/info/docs/MeshCtrlUsersGuide.pdf).

**Comment by kpetrow** [comment]
meshctrl was what i was looking for.  Thank you!
